src/pages/docs/kubernetes/targets/kubernetes-agent/upgrading.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Upgrading the agent
+navTitle: Overview
+navSection: Kubernetes agent
+description: How a Kubernetes Agent is upgraded
+navOrder: 10
+---
+
+Recent changes to the Kubernetes-Agent Helm Chart have necessitated a breaking change, meaning existing installations
+of the v1 Kubernetes-Agent helm chart are (currently) unable to automatically upgrade to thew new version (v2).
+
+Installed v1 instances will continue to operate as expected, however no further development work (bugfixes or enhancements)
```

Comment:
Should we give an instruction on how users can check the version of the agent they run?

Reply:
Done.